  config:
    joint_states_config:
      topic: joint_states
      channel_name: myRobotJointStates
      joint_names:
        - r_shoulder_pan_joint
        - r_shoulder_lift_joint
        - r_upper_arm_roll_joint
        - r_elbow_flex_joint
        - r_forearm_roll_joint
        - r_wrist_flex_joint
        - r_wrist_roll_joint
        - head_pan_joint
        - head_tilt_joint
      tolerances: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
      padding: .25
      min_samples: 5
      cache_size: 100
    pixel_configs:
    - pixel_topic: /stereo_narrow/left/led
      image_topic: /stereo_narrow/left/image_rect
      channel_name: LeftRect
      tolerance: 1.5
      padding: .5
      min_samples: 4
      cache_size: 100
    - pixel_topic: /stereo_narrow/right/led
      image_topic: /stereo_narrow/right/image_rect
      channel_name: RightRect
      tolerance: 1.5
      padding: .5
      min_samples: 4
      cache_size: 100
    joint_states_timeshift: .25
    pixel_timeshifts:
    - chan1: 0
      chan2: 1
      max_timeshift: 0.0

  commands:
  - [ 0.16423, -0.00795,  0.33131, -1.02525,  0.81061,  0.18658, -1.58223, -0.22206,  0.10031]
  - [ 0.16132, -0.01481,  0.33436, -1.02945,  0.81061,  0.41779, -1.57535, -0.22311,  0.10031]
  - [ 0.16132, -0.01810,  0.33597, -1.03770,  0.81570,  0.44359, -1.56295, -0.22384,  0.10031]
  - [ 0.16149, -0.02276,  0.33436, -1.04190,  0.81663,  0.56533, -1.57575, -0.22416,  0.10031]
  - [ 0.16166, -0.02783,  0.33709, -1.04291,  0.81767,  0.65605, -1.58223, -0.22437,  0.10031]
  - [ 0.16166, -0.03240,  0.34110, -1.04407,  0.81854,  0.75498, -1.58284, -0.22447,  0.10031]
  - [ 0.16157, -0.03629,  0.34110, -1.04537,  0.81900,  0.85179, -1.59245, -0.22468,  0.10020]
  - [ 0.16066, -0.04222,  0.35040, -1.04392,  0.81906,  0.99602, -1.60216, -0.22478,  0.10020]
  - [ 0.16083, -0.04585,  0.34896, -1.04392,  0.81906,  1.10645, -1.61904, -0.22478,  0.10020]
  - [ 0.16083, -0.05034,  0.34847, -1.04465,  0.81906,  1.27587, -1.63122, -0.22489,  0.10020]
  - [ 0.16000, -0.05414,  0.34944, -1.03886,  0.81906,  1.38578, -1.63192, -0.22499,  0.10020]
  - [ 0.22052,  0.14661,  0.35906, -1.11297,  0.81906,  0.63799, -1.64849, -0.22520,  0.10020]
  - [ 0.22035,  0.14018,  0.35826, -1.12803,  0.81906,  0.86263, -1.66550, -0.22499,  0.10020]
  - [ 0.21993,  0.13536,  0.36227, -1.13121,  0.81449,  0.96457, -1.66703, -0.22499,  0.10020]
  - [ 0.21985,  0.13367,  0.36018, -1.13628,  0.81246,  1.09871, -1.67473, -0.22499,  0.10020]
  - [ 0.21977,  0.13181,  0.36034, -1.13773,  0.80801,  1.17759, -1.68713, -0.22489,  0.10020]
  - [ 0.22027,  0.13198,  0.36114, -1.13860,  0.80309,  1.19103, -1.69161, -0.22468,  0.10020]
  - [ 0.03755,  0.03393,  0.29780, -1.20794,  0.67982,  0.84540, -1.60433, -0.22447,  0.10020]
  - [ 0.04560,  0.01920,  0.29235, -1.20866,  0.63331,  0.86110, -1.69818, -0.22499,  0.10020]
  - [ 0.03606,  0.00863,  0.29122, -1.21141,  0.62261,  0.79728, -1.79812, -0.22499,  0.10020]
  - [ 0.03789, -0.00330,  0.29074, -1.21445,  0.61891,  0.82229, -1.95393, -0.22499,  0.10020]
  - [ 0.03880, -0.01421,  0.29058, -1.21517,  0.61717,  0.82473, -2.11996, -0.22510,  0.10020]
  - [-0.03639, -0.02225,  0.31704, -1.20765,  0.61486,  0.87198, -2.16721, -0.22510,  0.10020]
  - [ 0.38035,  0.62385,  0.43876, -1.66393,  0.61665,  0.09930, -1.60581, -0.19044,  0.27823]
  - [ 0.36493,  0.62258,  0.39931, -1.73921,  0.66958,  0.09726, -1.60803, -0.18949,  0.27865]
  - [ 0.36186,  0.62360,  0.37093, -1.81448,  0.67785,  0.09674, -1.60933, -0.18939,  0.27865]
  - [ 0.36385,  0.62647,  0.34527, -1.92363,  0.68103,  0.09765, -1.60938, -0.18939,  0.27886]
  - [ 0.36567,  0.62935,  0.30934, -1.99688,  0.68653,  0.09813, -1.61151, -0.19002,  0.34357]
  - [ 0.36824,  0.59509,  0.27294, -2.02685,  0.68954,  0.09848, -1.61238, -0.19002,  0.45374]
  - [ 0.37322,  0.57791,  0.25306, -2.04422,  0.69185,  0.09870, -1.61260, -0.19002,  0.45500]
  - [ 0.33649,  0.31277,  0.32939, -1.73646,  0.29519,  0.19576, -1.60124, -0.18887,  0.27655]
  - [ 0.33517,  0.29450,  0.32747, -1.74123,  0.34714,  0.19681, -1.60055, -0.18887,  0.27655]
  - [ 0.33069,  0.27318,  0.32474, -1.74297,  0.52109,  0.19690, -1.60055, -0.18876,  0.27655]
  - [ 0.32174,  0.26159,  0.30598, -1.75035,  0.66374,  0.19703, -1.60050, -0.18876,  0.27655]
  - [ 0.31693,  0.25524,  0.30309, -1.76729,  0.76011,  0.19724, -1.60037, -0.18866,  0.27655]
  - [ 0.30474,  0.25067,  0.28337, -1.77887,  0.95164,  0.19724, -1.60037, -0.18866,  0.27655]
  - [ 0.29181,  0.24534,  0.28144, -1.78336,  1.09250,  0.19733, -1.60046, -0.18866,  0.27655]
  - [ 0.25815,  0.24171,  0.27567, -1.78452,  1.23729,  0.19755, -1.60068, -0.18845,  0.27655]
  - [ 0.24621,  0.23849,  0.27310, -1.78495,  1.34761,  0.19755, -1.60068, -0.18845,  0.27655]
  - [ 0.22383,  0.23621,  0.49681, -1.54508,  0.35582,  0.37750, -1.95379, -0.39967,  0.27613]
  - [ 0.22126,  0.23528,  0.40813, -1.54566,  0.32799,  0.38494, -1.96123, -0.18855,  0.27634]
  - [ 0.22309,  0.23748,  0.33468, -1.54552,  0.31903,  0.38760, -1.96319, -0.18855,  0.27634]
  - [ 0.21993,  0.23849,  0.28304, -1.54552,  0.31295,  0.39512, -1.96115, -0.18834,  0.27634]
  - [ 0.21935,  0.23985,  0.23349, -1.54248,  0.31047,  0.40526, -1.95458, -0.18824,  0.27634]
  - [ 0.22565,  0.24018,  0.21264, -1.54031,  0.30948,  0.44163, -1.92316, -0.18834,  0.27634]
  - [ 0.25409,  0.23731,  0.17047, -1.50180,  0.36571,  0.45155, -1.92534, -0.18824,  0.27634]
  - [ 0.27075,  0.23384,  0.06190, -1.47545,  0.40713,  0.45155, -1.92534, -0.18813,  0.27634]
  - [ 0.29015,  0.23155,  0.01315, -1.43622,  0.48869,  0.45416, -1.92595, -0.09996,  0.27488]
  - [ 0.29032,  0.23079,  0.01315, -1.43550,  0.49737,  0.45390, -1.92569,  0.24645,  0.27509]
  - [ 0.22085,  0.26116,  0.50708, -1.38310,  0.16110,  0.99294, -1.93870, -0.29955,  0.40463]
  - [ 0.20907,  0.26734,  0.42353, -1.37933,  0.11286,  0.97858, -1.93905, -0.18468,  0.27467]
  - [ 0.20120,  0.27715,  0.25113, -1.37745,  0.10812,  0.97675, -1.93896, -0.18478,  0.27467]
  - [ 0.19631,  0.27817,  0.18057, -1.37108,  0.10979,  0.97179, -1.93922, -0.18499,  0.27467]
  - [ 0.20170,  0.27792,  0.09141, -1.35024,  0.28843,  0.97170, -1.93922, -0.18489,  0.27467]
  - [ 0.20543,  0.27546,  0.00368, -1.30811,  0.40215,  0.95804, -1.92721, -0.18489,  0.27467]
  - [ 0.25011,  0.27902,  0.16020, -1.31723,  0.45167,  0.70047, -1.82166, -0.18478,  0.27467]
  - [-1.28595, -0.08198, -0.64195, -1.92595, -0.81726,  0.10157, -2.12200, -0.89887,  0.20650]
  - [-1.06220, -0.07386, -0.51719, -1.64584, -0.67982,  0.29831, -2.47991, -0.66691,  0.04428]
  - [-0.90361, -0.06658, -0.39852, -1.64844, -0.66113,  0.30606, -2.47825, -0.66723,  0.04408]
  - [-0.76890, -0.05321, -0.27680, -1.67407, -0.64899,  0.35235, -2.50810, -0.89866,  0.20524]
  - [-0.60070, -0.04315, -0.16871, -1.65525, -0.64291,  0.47405, -2.61204, -0.89876,  0.20503]
  - [-0.24804, -0.03638,  0.10616, -1.60530, -0.54908,  0.67610, -2.80705, -0.89866,  0.20503]
  - [-0.00597, -0.08629,  0.34944, -1.57042, -0.52392,  0.68707, -3.02216, -0.66712,  0.04334]
  - [ 0.32182, -0.13579,  0.49633, -1.62629, -0.07763,  0.84396, -3.02146, -0.70000,  0.20513]
  - [ 0.19565, -0.23062,  0.54957, -1.32085, -0.00087,  1.34514, -2.91517, -0.70000,  0.20482]
  - [-1.60520,  0.11650,  0.28609, -1.51888,  0.28073,  0.15360, -1.54733, -1.69934,  0.15372]
  - [-1.60611,  0.11193,  0.28577, -1.52019,  0.28646,  0.16117, -1.54533, -1.69819,  0.15330]
  - [-1.60852,  0.11108,  0.21906, -1.53958,  0.28918,  0.17057, -1.54290, -1.69798,  0.15319]
  - [-1.61009,  0.10964,  0.14529, -1.54060,  0.29109,  0.17649, -1.54255, -1.50132,  0.00093]
  - [-1.61523,  0.10736,  0.04169, -1.54132,  0.29172,  0.17914, -1.54198, -1.50132,  0.00093]
  - [-1.61374,  0.10398, -0.05341, -1.54031,  0.29242,  0.18197, -1.54246, -1.50121,  0.00093]
  - [-1.61274,  0.10220, -0.10504, -1.54132,  0.29352,  0.18719, -1.54290, -1.50121,  0.00093]
  - [-1.60910,  0.09848, -0.17079, -1.54335,  0.29392,  0.19002, -1.54285, -1.50121,  0.00093]
  - [-1.60213,  0.09467, -0.23382, -1.54653,  0.29433,  0.19320, -1.54264, -1.50132,  0.00093]
  - [-1.59815,  0.09035, -0.27744, -1.55044,  0.29479,  0.19603, -1.54320, -1.50132,  0.00093]
  - [-1.59426,  0.08596, -0.32876, -1.55580,  0.29514,  0.19864, -1.54355, -1.50132,  0.00083]
  - [-1.57254,  0.08858, -0.28177, -1.56984,  0.24614,  0.25541, -1.26888, -1.50132,  0.00083]
  - [-2.13037, -0.13638,  0.55551, -1.34170,  0.28987,  0.16996, -1.52671, -2.29792,  0.04072]
  - [-2.13220, -0.14010,  0.48094, -1.34039,  0.29224,  0.17336, -1.52784, -2.29687,  0.03936]
  - [-2.13170, -0.14255,  0.41118, -1.33648,  0.29433,  0.17619, -1.52945, -2.13299, -0.21961]
  - [-2.13220, -0.14518,  0.34879, -1.33055,  0.29485,  0.17784, -1.53041, -2.13341, -0.22003]
  - [-2.13178, -0.14645,  0.26973, -1.32867,  0.29496,  0.17993, -1.53102, -2.13362, -0.22024]
  - [-2.12971, -0.14780,  0.19821, -1.32794,  0.29502,  0.18202, -1.53145, -2.13341, -0.22034]
  - [-2.11976, -0.15076,  0.15796, -1.32751,  0.29514,  0.18306, -1.53198, -2.13414, -0.22055]
  - [-2.10351, -0.15330,  0.11338, -1.33055,  0.29519,  0.18423, -1.53245, -2.13435, -0.22076]
  - [-1.77622,  0.45296,  0.51173, -1.75441,  0.29629,  0.24193, -1.66637, -1.94826,  0.35268]
  - [-1.78111,  0.44458,  0.43620, -1.75614,  0.29641,  0.24193, -1.66637, -1.94847,  0.35038]
  - [-1.77995,  0.44069,  0.36868, -1.75643,  0.29652,  0.24193, -1.66637, -1.71736,  0.12544]
  - [-1.77995,  0.43629,  0.30854, -1.75716,  0.29664,  0.24193, -1.66637, -1.71736,  0.12544]
  - [-1.77506,  0.43350,  0.23638, -1.76063,  0.29681,  0.24193, -1.66637, -1.71736,  0.12544]
  - [-1.76462,  0.42876,  0.17816, -1.76280,  0.29704,  0.24354, -1.66616, -1.71756,  0.12534]
  - [-1.76429,  0.41658,  0.06382, -1.76309,  0.29710,  0.26834, -1.64754, -1.71736,  0.12534]
  - [-1.76702,  0.40820, -0.00417, -1.77106,  0.29728,  0.36018, -1.57405, -1.71736,  0.12534]
  - [-1.78128,  0.39737, -0.01877, -1.78394,  0.42500,  0.53948, -1.23629, -1.71756,  0.12534]
  - [-1.77639,  0.38502, -0.13247, -1.79509,  0.44403,  0.57555, -1.19952, -1.71788,  0.12523]
  - [-1.74522,  0.38240, -0.05998, -1.80913,  0.57633,  0.56076, -1.14314, -1.71788,  0.12523]
  - [-1.77382,  1.05921,  0.36435, -1.96880,  0.50616,  0.39560, -1.44065, -1.61023,  0.51647]
  - [-1.76735,  1.07825,  0.31207, -2.01063,  0.42136,  0.40643, -1.45636, -1.60970,  0.51647]
  - [-1.77274,  1.08535,  0.25434, -2.02178,  0.40464,  0.41461, -1.46280, -1.60918,  0.51668]
  - [-1.77299,  1.08755,  0.21874, -2.02410,  0.39891,  0.41757, -1.46445, -1.60928,  0.51688]
  - [-1.77150,  1.09060,  0.13535, -2.03886,  0.39637,  0.41727, -1.46676, -1.60918,  0.51688]
  - [-1.77266,  1.09178,  0.06751, -2.04161,  0.39498,  0.41796, -1.46797, -1.60918,  0.51688]
  - [-1.78120,  1.09229, -0.01091, -2.04320,  0.39423,  0.41862, -1.46871, -1.60908,  0.51688]
  - [-1.79645,  1.08544, -0.13712, -2.04393,  0.39394,  0.41857, -1.46876, -1.60897,  0.51699]
  - [-1.80060,  1.07283, -0.24328, -2.04393,  0.39348,  0.41857, -1.46876, -1.60897,  0.51709]
  - [-1.84387,  1.04610, -0.43989, -2.04407,  0.39405,  0.30893, -1.38705, -1.60887,  0.51709]
  - [-1.84901,  1.03375, -0.52777, -2.04509,  0.39353,  0.30902, -1.38905, -1.60887,  0.51709]
  - [-1.85357,  1.02283, -0.59128, -2.04538,  0.39319,  0.30980, -1.39062, -1.60876,  0.51730]
  - [-1.85465,  1.01699, -0.62143, -2.04523,  0.39215,  0.31058, -1.39166, -1.60887,  0.51741]
  - [-1.73577,  0.48417,  0.36419, -0.98283,  0.39162,  0.81951, -1.76283, -1.60803,  0.51751]
  - [-1.73229,  0.47038,  0.33067, -1.14352,  0.35559,  0.84440, -1.74682, -1.60813,  0.51751]
  - [-1.73129,  0.47055,  0.32666, -1.24123,  0.34506,  0.82812, -1.72855, -1.60792,  0.51751]
  - [-1.73096,  0.47241,  0.31881, -1.33952,  0.33881,  0.83626, -1.72668, -1.60792,  0.51751]
  - [-1.73071,  0.47664,  0.30566, -1.46836,  0.33881,  0.52795, -1.76705, -1.60792,  0.51751]
  - [-1.73030,  0.48037,  0.30501, -1.54219,  0.33505,  0.60636, -1.70858, -1.60782,  0.51751]
  - [-1.72980,  0.48527,  0.29507, -1.65481,  0.33262,  0.61206, -1.71584, -1.60803,  0.51751]
  - [-1.72789,  0.49678,  0.28401, -1.83070,  0.33262,  0.41601, -1.68835, -1.60803,  0.51751]
  - [-1.72756,  0.50152,  0.28946, -1.95027,  0.33175,  0.36162, -1.69044, -1.60803,  0.51751]
  - [-1.72673,  0.50346,  0.29058, -2.04697,  0.33152,  0.23906, -1.69466, -1.60813,  0.51751]
  - [-1.77191,  0.48883, -1.01192, -1.88324, -0.48846,  0.46452, -2.08589, -1.18308,  0.47981]
  - [-1.78783,  0.47631, -0.92709, -2.00195, -0.51738,  0.23666, -2.01493, -1.18266,  0.47971]
  - [-1.79827,  0.47478, -0.83295, -2.04682, -0.54018,  0.22883, -2.01292, -1.18203,  0.47940]
  - [-1.56847,  0.35913, -1.19666, -1.32360, -0.53948,  0.81494, -2.17760, -1.18171,  0.47919]
  - [-1.59451,  0.34788, -1.04608, -1.50441, -0.62527,  0.61889, -2.17613, -1.18161,  0.47866]
  - [-1.61241,  0.34797, -0.96702, -1.63310, -0.65934,  0.57486, -2.22372, -1.18182,  0.47803]
  - [-1.63148,  0.34704, -0.87705, -1.78119, -0.66669,  0.47879, -2.22529, -1.18119,  0.47761]
  - [-1.64640,  0.34797, -0.82541, -1.90250, -0.67496,  0.40870, -2.22594, -1.18119,  0.47709]
  - [ 0.55891,  1.12740, -0.66184, -1.46156,  0.01550,  0.59870, -1.85764,  0.89886,  0.94885]
  - [ 0.56240,  1.15456, -0.65414, -1.57867,  0.06369,  0.52582, -1.80408,  0.89917,  0.94885]
  - [ 0.56231,  1.15769, -0.62624, -1.68970,  0.07231,  0.52430, -1.80291,  0.89907,  0.94885]
  - [ 0.56273,  1.16006, -0.61629, -1.78741,  0.07456,  0.52469, -1.80217,  0.89907,  0.94885]
  - [ 0.56289,  1.16124, -0.61196, -1.87239,  0.07543,  0.53022, -1.80117,  0.89896,  0.94885]
  - [ 0.56289,  1.16175, -0.60876, -1.91972,  0.07653,  0.53239, -1.79986,  0.89886,  0.94885]
  - [ 0.56206,  0.94009, -0.37061, -1.45823,  0.07902,  0.66605, -2.07971,  0.59999,  0.94802]
  - [ 0.56405,  0.92918, -0.47389, -1.45895,  0.05339,  0.64713, -2.11012,  0.89875,  0.94802]
  - [ 0.56289,  0.91793, -0.59529, -1.44940,  0.05334,  0.86136, -2.00735,  0.89875,  0.94802]
  - [ 0.56347,  0.92596, -0.75293, -1.44940, -0.01059,  0.92550, -1.87691,  0.89896,  0.94802]
  - [ 0.56347,  0.92766, -0.75421, -1.44795, -0.01776,  0.92023, -1.88105,  0.89896,  0.94802]
  - [ 0.56397,  0.95650, -0.82798, -1.44809, -0.01776,  0.86106, -1.57483,  0.89907,  0.94802]
  - [ 0.56414,  0.96996, -0.89758, -1.43999,  0.06508,  0.85836, -1.59928,  0.89928,  0.94802]
  - [-0.12891, -0.19780,  0.15299, -1.26092,  0.48667,  0.62546, -2.00361, -0.48889, -0.10285]
  - [ 0.41881,  0.08367,  0.29363, -1.29523,  0.49077,  0.39791, -1.69470,  0.00036,  0.00386]
  - [ 0.41832,  0.07436,  0.29507, -1.30232,  0.50911,  0.39791, -1.69470,  0.01869,  0.00397]
  - [ 0.41790,  0.06785,  0.29587, -1.30319,  0.51461,  0.39804, -1.69457,  0.04309,  0.00386]
  - [ 0.41790,  0.06785,  0.29587, -1.30319,  0.51461,  0.39804, -1.69457,  0.04309,  0.00386]
  - [ 0.41765,  0.06320,  0.29619, -1.30348,  0.51750,  0.39817, -1.69444,  0.07000,  0.00386]
  - [ 0.41765,  0.05998,  0.29619, -1.30362,  0.51918,  0.39839, -1.69422,  0.09608,  0.00386]
  - [ 0.41757,  0.05897,  0.29635, -1.30362,  0.52022,  0.39865, -1.69413,  0.12330,  0.00376]
  - [ 0.41757,  0.05846,  0.29652, -1.30377,  0.52091,  0.39882, -1.69413,  0.14509,  0.00376]
  - [ 0.41757,  0.05728,  0.29668, -1.30391,  0.52224,  0.39965, -1.69409, -0.19567,  0.00365]
  - [ 0.41765,  0.05719,  0.29700, -1.30420,  0.52253,  0.39986, -1.69413, -0.17389,  0.00365]
  - [ 0.41765,  0.05719,  0.29716, -1.30420,  0.52282,  0.40013, -1.69413, -0.14499,  0.00355]
  - [ 0.41765,  0.05719,  0.29716, -1.30435,  0.52305,  0.40034, -1.69400, -0.12342,  0.00355]
  - [ 0.41774,  0.05711,  0.29716, -1.30449,  0.52323,  0.40060, -1.69400, -0.10090,  0.00355]
  - [ 0.41765,  0.05694,  0.29732, -1.30449,  0.52340,  0.40087, -1.69400, -0.07409,  0.00355]
  - [ 0.41782,  0.05719,  0.29716, -1.30464,  0.52352,  0.40117, -1.69396, -0.04708,  0.00355]
  - [ 0.41765,  0.05711,  0.29732, -1.30478,  0.52363,  0.40143, -1.69387, -0.00205,  0.00344]
  - [ 0.41774,  0.05711,  0.29732, -1.30478,  0.52381,  0.40208, -1.69374, -0.00152,  0.26797]
  - [ 0.41765,  0.05685,  0.29748, -1.30493,  0.52386,  0.40226, -1.69366, -0.00131,  0.26482]
  - [ 0.41765,  0.05685,  0.29748, -1.30507,  0.52398,  0.40234, -1.69366, -0.00173,  0.20597]
  - [ 0.41774,  0.05694,  0.29748, -1.30507,  0.52409,  0.40248, -1.69361, -0.00173,  0.19697]
  - [ 0.41774,  0.05711,  0.29748, -1.30522,  0.52421,  0.40256, -1.69361, -0.00173,  0.14974]
  - [ 0.41774,  0.05685,  0.29748, -1.30522,  0.52427,  0.40269, -1.69357, -0.00152,  0.13162]
  - [ 0.41765,  0.05677,  0.29748, -1.30536,  0.52433,  0.40278, -1.69348, -0.00152,  0.11350]
  - [ 0.41765,  0.05677,  0.29748, -1.30536,  0.52444,  0.40287, -1.69348, -0.00131,  0.07392]
  - [ 0.41765,  0.05677,  0.29764, -1.30536,  0.52450,  0.40304, -1.69339, -0.00142,  0.05570]
  - [ 0.41765,  0.05677,  0.29764, -1.30551,  0.52456,  0.40317, -1.69335, -0.00131,  0.03517]
  - [ 0.41765,  0.05677,  0.29764, -1.30551,  0.52462,  0.40326, -1.69326, -0.00142,  0.00554]
  - [-0.15875, -0.03926, -0.70177, -1.29870,  0.53798,  0.47931, -1.69492,  0.11660,  0.10764]
  - [-0.15801, -0.04526, -0.70963, -1.30232,  0.53844,  0.46404, -1.69383,  0.11754,  0.20032]
  - [-0.24290, -0.02386, -0.86101, -1.30623,  0.36264,  0.71213, -1.03006,  0.15420,  0.20210]
  - [-0.42735, -0.02978, -1.24157, -1.45490,  0.22040,  0.55128, -0.68586,  0.05995,  0.50390]
  - [-0.52907, -0.12056, -1.91783, -1.74630,  0.22022,  0.56485, -0.74338,  0.01764,  0.80832]
  - [ 0.42752, -0.02343,  0.62254, -1.42218,  0.88437,  0.55684, -1.52297, -0.24751, -0.00399]
  - [ 0.25003,  0.40296,  0.44822, -1.67233,  0.57743,  0.21530, -1.47067, -0.35024,  0.24147]
  - [ 0.19308,  0.69822,  0.63184, -1.64207, -0.18280,  0.84187, -1.93600, -0.58450,  0.46285]
  - [ 0.23328,  0.82622,  0.14481, -1.05116, -0.18054,  1.03727, -1.88157, -0.11211,  0.52725]
  - [ 0.24820,  0.68400,  0.13647, -1.42899, -0.17487,  0.44655, -1.90650, -0.11305,  0.52746]
  - [ 0.26246,  0.38857,  0.14625, -1.37383, -0.12437,  0.25333, -2.00357, -0.14007,  0.20482]
  - [ 0.22491, -0.02800,  0.12861, -1.12745, -0.12437,  0.27425, -2.00344, -0.14164,  0.02994]
  - [ 0.25202, -0.14974,  0.07810, -1.06578,  0.01753,  0.28509, -2.00827, -0.09996, -0.19688]
  - [-0.17832, -0.29543, -1.16443, -0.98964, -0.22717,  1.17502, -1.90654,  0.06005,  0.39300]
  - [-0.23096, -0.14543, -1.61105, -1.16596, -0.22589,  1.12459, -1.87874,  0.17001,  0.61009]
  - [-0.25964, -0.18172, -2.43021, -1.59054, -0.19494,  0.71165, -1.46145,  0.08498,  0.94770]
  - [-0.37305,  0.02225, -2.48762, -1.31520, -0.17620,  0.83656, -1.27597, -0.15483,  0.94770]
  - [-1.60595, -0.13113, -1.31229, -1.68492,  0.09232,  0.87285, -0.92137, -0.92997,  0.40693]
  - [-1.52744,  0.19526, -2.27545, -1.76715,  0.09232,  0.35618, -1.42977, -0.92997,  0.91419]
  - [-0.95750,  0.02090, -3.75099, -0.82273,  0.63105,  1.22553, -2.10038, -1.34413,  0.64464]
  - [-0.33160,  0.04535, -3.75003, -0.61210,  0.62949,  1.24520, -2.09750, -0.82954,  0.64454]
  - [-0.97947, -0.23638, -3.75019, -0.19490,  0.91815,  1.77135, -2.43953, -1.23093,  0.35457]
  - [-0.43921,  0.45668, -3.75051, -0.66986,  0.64395,  1.34096, -1.83301, -1.10715,  0.81062]
  - [ 0.55096,  0.49221, -3.75003, -0.04609,  0.64291,  1.40275, -2.29917,  0.05461,  0.94686]
  - [ 0.37753, -0.09729, -3.74955, -0.03929,  1.00538,  1.69086, -2.02750, -0.09996,  0.51113]
  - [ 0.37065, -0.27690, -3.75067, -0.85487,  0.94869,  1.65196, -2.06674, -0.20007,  0.60956]
  - [ 0.56265, -0.05313, -0.87000, -1.02409,  0.13982,  0.59927, -1.49943,  0.92001,  0.30598]
  - [ 0.56414,  0.29188, -1.54258, -0.94939,  0.13276,  0.78744, -1.17490,  1.01991,  0.81293]
  - [ 0.52808,  0.50879, -1.56904, -0.72661,  0.43883,  1.27870, -1.01988,  0.89205,  0.94425]
  - [ 0.56397, -0.20634, -0.42257, -0.83706,  0.16891,  0.65143, -1.82475,  0.70000, -0.09677]
  - [-0.21164, -0.07758, -0.35714, -1.21329,  0.16851,  0.80711, -1.27445, -0.15001,  0.13057]
  - [ 0.21969, -0.06929,  0.18714, -1.15206,  0.18187,  0.70534, -1.93826, -0.15253, -0.13374]
  - [ 0.33309, -0.12225,  0.44309, -1.08185,  0.37716,  0.93729, -2.25079, -0.15232, -0.13384]
  - [ 0.34561, -0.10795,  0.41679, -1.11457,  0.39226,  0.94829, -2.27267, -0.25002,  0.13036]
  - [ 0.25915,  0.02597,  0.37814, -1.21706,  0.39313,  0.87433, -2.12709, -0.18436, -0.01404]
  - [ 0.31253,  0.17665,  0.34575, -1.26989,  0.39301,  0.87694, -2.07488, -0.20007,  0.23529]
  - [ 0.31635,  0.18198,  0.35104, -1.27279,  0.40476,  0.86920, -2.07166, -0.00278,  0.46850]
  - [ 0.31701,  0.18257,  0.35345, -1.27293,  0.40817,  0.86384, -2.07101, -0.00142,  0.46850]
  - [ 0.31726,  0.18299,  0.35441, -1.27293,  0.41002,  0.86002, -2.07101, -0.00100,  0.46631]
  - [ 0.31743,  0.18333,  0.35489, -1.27293,  0.41129,  0.85710, -2.07114, -0.00121,  0.43719]
  - [ 0.31759,  0.18350,  0.35553, -1.27293,  0.41233,  0.85506, -2.07127, -0.00131,  0.42232]
  - [ 0.31776,  0.18375,  0.35569, -1.27279,  0.41314,  0.85340, -2.07136, -0.00142,  0.39342]
  - [ 0.31801,  0.18401,  0.35585, -1.27279,  0.41372,  0.85201, -2.07144, -0.00131,  0.35698]
  - [-0.11407,  0.19619, -0.18378, -1.54870,  1.11894,  1.23841, -0.09348, -0.00089,  0.35593]
  - [ 0.13745,  0.19078,  0.14481, -1.51208,  1.15145,  1.44190, -0.19516, -0.00048,  0.35541]
  - [ 0.13745,  0.19078,  0.14481, -1.51208,  1.15145,  1.44190, -0.19516, -0.00048,  0.35541]
  - [-0.53123, -0.02699, -0.44534, -1.36732,  0.65599,  0.65700, -0.77862, -0.22447, -0.01687]
  - [-0.05355,  0.26252,  0.32458, -1.25079,  0.61578,  0.71744, -1.56443, -0.45004,  0.20095]
  - [ 0.08016,  0.57994,  0.35697, -1.31709,  0.60878,  0.94581, -1.53889, -0.32930,  0.38358]
  - [-1.62070,  0.14585,  0.31688, -1.45215,  0.52999,  0.97236, -1.49643, -1.69997,  0.10460]
  - [-1.55372,  0.13849,  0.28144, -1.45678,  0.52913,  0.97779, -1.49073, -1.69840,  0.10439]
  - [-1.50348,  0.13215,  0.27246, -1.46199,  0.52872,  0.98114, -1.48729, -1.49996,  0.10460]
  - [-1.42970,  0.12513,  0.26524, -1.46880,  0.52838,  0.98319, -1.48577, -1.59996,  0.10439]
  - [-1.34489,  0.11929,  0.25706, -1.47010,  0.52785,  0.99694, -1.48673, -1.49996,  0.10387]
  - [-0.98801,  0.09619,  0.31736, -1.37296,  0.52785,  0.94521, -1.69309, -1.16004,  0.10240]
  - [-0.68459,  0.09628,  0.26653, -1.40351,  0.52479,  0.94542, -1.69287, -1.00023,  0.10251]
  - [-0.46350,  0.09120,  0.24921, -1.40380,  0.52433,  0.94547, -1.69283, -0.80012,  0.10230]
  - [-0.28634,  0.06887,  0.23060, -1.39265,  0.52433,  0.87829, -1.82988, -0.66995,  0.10209]
  - [ 0.08323,  0.05457,  0.26027, -1.30551,  0.52433,  0.77078, -2.17504, -0.35401, -0.05174]
  - [ 0.33210,  0.05085,  0.27823, -1.32939,  0.52103,  0.83896, -1.90759, -0.09996,  0.10083]
  - [ 0.47684,  0.01777,  0.23044, -1.32143,  0.52080,  0.47657, -1.73255,  0.07681, -0.11164]
  - [ 0.36650,  0.06100,  0.43876, -1.44505, -0.03008,  1.49616, -3.21551, -0.35998,  0.39782]
  - [-0.05405,  0.05922,  0.50130, -1.48964, -0.02748,  1.38726, -3.05953, -0.71006,  0.39855]
  - [-2.12739,  0.07318,  0.39674, -1.46431, -0.17898,  0.60627, -2.68770, -2.09717, -0.02661]
  - [-2.12739,  0.06277,  0.25995, -1.44998, -0.19188,  0.64595, -2.40794, -2.09969, -0.02661]
  - [-2.10749,  0.05398,  0.15395, -1.42566, -0.05565,  0.37189, -2.01971, -2.10011, -0.02672]
  - [-2.10625,  0.04323, -0.09077, -1.39526, -0.03459,  0.14638, -2.14637, -1.79506, -0.02022]
  - [-2.12805,  0.04839, -0.36420, -1.45779, -0.03419,  0.30188, -1.58849, -1.79558, -0.03677]
  - [-2.09796,  0.13359, -0.50195, -1.51541, -0.02835,  0.36079, -1.66168, -1.59033, -0.03677]
  - [-2.13137,  0.33375, -0.34271, -1.50368,  0.20212,  0.97418, -1.04629, -1.65798,  0.12534]
  - [-1.63148,  0.14272,  0.33965, -1.54624,  0.52664,  0.67049, -1.34867, -1.76144, -0.03687]
  - [ 0.13148,  0.13562,  0.26284, -1.66972,  1.99474,  0.42810, -0.39265, -0.32532, -0.03645]
  - [ 0.16903,  0.10059,  0.28962, -1.63831,  2.01233,  1.19369, -0.36045, -0.10006,  0.29195]
  - [ 0.13223,  0.08291,  0.28673, -1.62832,  2.02974,  0.83774, -0.62338, -0.10792, -0.00451]
  - [ 0.14806,  0.07039,  0.28096, -1.61906,  2.04634,  0.89378, -0.43090, -0.10834, -0.00514]
  - [ 0.14499,  0.06193,  0.28192, -1.59705,  2.04761,  0.89082, -0.21692, -0.10855, -0.00514]
  - [ 0.14640,  0.05728,  0.28593, -1.59257,  2.05010,  1.05037, -0.02161, -0.10865, -0.00514]
  - [ 0.15055,  0.05575,  0.29219, -1.57288,  2.05461,  1.00342,  0.04814, -0.10876, -0.00525]
  - [-0.06342,  0.21294,  0.23606, -1.59662, -0.47035,  1.60297,  2.83489, -0.69990,  0.55741]
  - [-0.06226,  0.21252,  0.25354, -1.59474, -0.47093,  1.48154,  2.66246, -0.69990,  0.55741]
  - [-0.06151,  0.21235,  0.26220, -1.59213, -0.47336,  1.55677,  2.56078, -0.69990,  0.55741]
  - [-0.05720,  0.21210,  0.26540, -1.58779, -0.47365,  1.58018,  2.42642, -0.70000,  0.55741]
  - [-0.05463,  0.21167,  0.26893, -1.57519, -0.47365,  1.58696,  2.23725, -0.70011,  0.55741]
  - [-0.05289,  0.21117,  0.26589, -1.56709, -0.47874,  1.60267,  2.24095, -0.70074,  0.55741]
  - [-0.08025,  0.01675,  0.06687, -1.22241, -0.80755,  1.53092,  2.69531, -0.67005,  0.20262]
  - [-0.09401, -0.00144,  0.02213, -1.23009, -0.73639,  1.58553,  2.64184, -0.66995,  0.20210]
  - [-0.09542, -0.01988,  0.00994, -1.22328, -0.58958,  1.58561,  2.64192, -0.67351,  0.02261]
  - [-0.27440, -0.04636,  0.14769, -1.32476, -0.56650,  1.43929,  2.80121, -0.63382,  0.02261]
  - [-0.27705, -0.06387,  0.08884, -1.31651, -0.37589,  1.53158,  2.71319, -0.63214,  0.02261]
  - [-0.28849, -0.07394,  0.07072, -1.31795, -0.22572,  1.53162,  2.71341, -0.63099,  0.02250]
  - [-0.82038, -0.14298,  0.19901, -1.21648, -0.22936,  1.48533,  2.87996, -1.09972,  0.09602]
  - [-0.83215, -0.15144,  0.13454, -1.22285, -0.22884,  1.49816,  2.87452, -1.09982,  0.09706]
  - [-0.12319, -0.17183,  0.20479, -1.14641,  1.16059,  1.52296, -0.46975, -0.29547, -0.03520]
  - [-1.50862, -0.17056,  0.26316, -1.22676,  1.27240,  1.43037, -0.37707, -1.30005,  0.14209]
  - [-1.49867, -0.17902,  0.27471, -1.22676,  1.22786,  1.34218, -0.31325, -1.38498, -0.19154]
  - [-1.48632, -0.19044,  0.28401, -1.22343,  1.04258,  1.34179, -0.31364, -1.38571, -0.19165]
  - [-1.50033, -0.19568,  0.26861, -1.26020,  1.04240,  0.84087, -0.78758, -1.38571, -0.19175]
